docs/configuration/tools/router_check.rst

```diff
@@ -0,0 +1,98 @@
+.. _config_tools_router_check_tool:
+
+Router check tool input interface
+=================================
+
+Input for the router check tool. The router check tool checks if the route returned
+by a router matches what is expected. The expected field specifies the expected value
+and is required. The authority and path fields specifies the url sent to the router
+and are also required. The simplest configuration has one test case and takes the following form: ::
+
+   {
+     "input": [
+       { "expected":"locations",
+         "authority":"api.lyft.com",
+         "path": "/api/locations"
+       }
+     ]
+   }
+
+.. code-block:: json
+
+  {
+    "input": {
+      "type" : "array",
+      "items" : {
+        "type": "object",
+        "properties": {
+          "expected": { "type": "string" },
+          "authority": { "type": "string" },
+          "path": { "type": "string" },
+          "additional_headers": {
+            "type": "array",
+            "items" : {
+              "type": "object",
+             "properties": {
+                "name": { "type": "string" },
+                "value": { "type": "string" }
+               }
+            }
+          },
+          "check": {
+            "type": "object",
+            "properties" : {
+              "name": {"type" : "string", "enum" : ["cluster", "virtual_cluster", "virtual_host"] },
```

Comment:
As discussed, the expected/checked properties can be reorganized so that individual input entries can be more expressive.